### PR TITLE
Add Datadog exporter configuration

### PR DESCRIPTION
**Pull request by shigeya-dd**
## Changes

Add Datadog exporter configuration for Open Telemetry Collector
in `src/otelcollector/otelcol-config-extras.yml `

### CODE REVIEW

**Comment by shigeya-dd** [comment]
Please review this PR. Thanks!
@dineshg13

File: src/otelcollector/otelcol-config-extras.yml [hunk 1]
```diff
@@ -1,2 +1,26 @@
 # extra settings to be merged into OpenTelemetry Collector configuration
 # do not delete this file
+
+exporters:
+  datadog:
+    api:
+      key: <DATADOG_API_KEY> # Replace with your Datadog API key
```
Comment (dineshg13) [inline src/otelcollector/otelcol-config-extras.yml:7]: Would prefer `DD_API_KEY`. We use that in most examples

**Review by dineshg13 — COMMENTED** [review]
@shigeya-dd FYI we have a configuration for helm-charts of demo , and collector config here https://github.com/DataDog/opentelemetry-helm-charts/blob/main/charts/opentelemetry-demo-datadog/otelcol-config.yaml

File: src/otelcollector/otelcol-config-extras.yml (hunk 1 above)
Comment (shigeya-dd) [reply src/otelcollector/otelcol-config-extras.yml]: Thanks! Fixed.

**Comment by shigeya-dd** [comment]
Preventing auto closing.
@dineshg13 Would you please review this PR?

**Review by dineshg13 — APPROVED** [review]

**Pull request merged**